storage/api/src/test/java/org/apache/kafka/server/log/remote/storage/NoOpRemoteLogMetadataManager.java

```diff
@@ -79,6 +79,11 @@ public long remoteLogSize(TopicIdPartition topicIdPartition, int leaderEpoch) {
         return 0;
     }
 
+    @Override
+    public boolean isInitialized(TopicIdPartition topicIdPartition) {
```

Comment:
This is not needed as there is a default implementation in `RemoteLogMetadataManager`.